backend/tslint.json

```diff
@@ -59,7 +59,8 @@
             "dir",
             "warn",
             "error"
-        ]
+        ],
+        "no-unused-variable": true
```

Comment:
Eventuell wäre es sogar sinnvoll hier noch 2 weitere Linting-Rules einzufügen: 
- https://eslint.org/docs/rules/valid-jsdoc
- https://eslint.org/docs/rules/require-jsdoc

Das erstere überprüft einfach, ob ein angegebenes Js-Doc auch zur Funktion/Klasse mit ihren Parametern passt. Damit filtern wir dann Copy-Paste-Fehler in den Js-Docs gleich vor dem PR heraus. Und dass das JsDoc in manchen Situationen fehlt, merken wir wir ja auch oft genug bei PRs an.

Reply:
Guter Vorschlag, ich schreibe mir das auf und probiere das bei Gelegenheit. Allerdings nutzen wir tslint (und nicht eslint). Wir müssen prüfen inwieweit sich das kombinieren lässt!

Reply:
Stimmt natürlich. Aber es gibt dieses Package genau dafür:

- https://www.npmjs.com/package/tslint-eslint-rules

Dort wird valid-jsdoc unterstützt, steht dabei. Was leider nicht unterstützt wird ist require-jsdoc. Dazu habe ich aber gerade dieses Paket gefunden:

- https://www.npmjs.com/package/tslint-jsdoc-rules

Eventuell schaue ich mir das auch nochmal am Ende der Woche an, wenn ich wieder Zeit habe.